app/views/best.html

```diff
@@ -98,29 +98,17 @@ <h1 translate="LB"></h1>
           </label>
         </p>
       </div></span>
-      <span><ul ng-show="normal.showHelp">
-        <li>Lock a set of perks by clicking a lock bucket and selecting perks</li>
-        <ul>
-          <li>If a perk doesn't appear it means that you own no armor with that perk</li>
-          <li>To use armor with multiple perks together shift+click the desired perks</li>
-          <li><div class="example ex-or"></div> - Armor with any of these perks will be used ("or")</li>
-          <li><div class="example ex-and"></div> - Armor with all of these perks will be used ("and")</li>
-        </ul>
-        <li>Drag and drop items into the locked buckets to build sets with only that gear</li>
-        <li>Shift click an item to build sets without that gear</li>
-        <li>Higher Tiers are better</li>
-        <li>
-          <ul>
-            <li>4/5/2 (a Tier 11 build) is 4 Intellect, 5 Discipline, 2 Strength (4+5+2 = Tier 11)</li>
-            <li>Intellect speeds up Super recharge time</li>
-            <li>Discipline speeds up Grenade recharge time</li>
-            <li>Strength speeds up Melee recharge time</li>
-          </ul>
-        </li>
-        <li>Try to find armor that has ammo increasing perks for weapon types that you use.</li>
-        <li>Change the Intellect, Discipline, or Strength nodes in game to what is displayed to create each loadout.</li>
-        <li>As an items defense level increases, the stats on that item (int/dis/str) also increase.</li>
-      </ul></span>
+      <span><div ng-show="normal.showHelp">
+        <iframe ng-show="normal.lang === 'en'" src="/views/en/best-help.html" width="80%" height="280em" scrolling="no" style="border:none" name="showHelp" id="showHelp"></iframe>
+        <!--
+          <iframe ng-show="normal.lang === 'de'" src="/views/de/best-help.html" width="80%" height="280em" scrolling="no" style="border:none" name="showHelp" id="showHelp"></iframe>
```

Comment:
Also, the iframe is a cool idea, but why not just load the right file directly into the template?